src/mercury_engine_data_structures/formats/brfld.py

```diff
@@ -11,22 +12,30 @@
 logger = logging.getLogger(__name__)
 
 
+class ActorLayer(Enum):
+    ENTITIES = auto
```

Comment:
Personal preference: use actual numbers instead of `auto` here

Reply:
`auto` is specifically for when the underlying values do not matter, which I don't believe applies in this case since these should be strings anyway?

Reply:
Yeah, I guess that was a bit of autopilot on my part (no pun intended). Those should definitely be strings.